projects/components/src/quick-search/quick-search.component.html

```diff
@@ -11,10 +11,16 @@
             *ngFor="let item of searchSection.result?.items"
             (click)="itemClicked(item)"
             [ngClass]="item == selectedItem && !this.currentFilterOptionSelection ? 'selected' : ''"
+            [attr.data-ui]="DataUi.searchResultItem"
         >
             {{ item.displayText }}
         </li>
-        <li class="no-results" data-ui="no-results" *ngIf="showNoResults(searchSection)">
+        <li
```

Comment:
This has been changed to a single no results section. Just something to be aware of when you rebase.